azure-devops/azext_devops/dev/artifacts/universal.py

```diff
@@ -45,7 +45,7 @@ def download_package(feed, name, version, path, file_filter=None, organization=N
     :type version: str
     :param path: Directory to place the package contents.
     :type path: str
-    :param file_filter: File filter.
+    :param file-filter: Wildcard filter for file download.
```

Comment:
it should be `file_filter` that is why linting is failing.

cc: @vinitsuri-msft